src/xc_integrator/shell_batched/shell_batched_replicated_xc_integrator_exx.hpp

```diff
@@ -1,5 +1,5 @@
 /**
- * GauXC Copyright (c) 2020-2024, The Regents of the University of California,
+ * GauXC Copyright (c) 2020-2023, The Regents of the University of California,
```

Comment:
I'll only note this a few times, just make sure it gets propagated
```suggestion
 * GauXC Copyright (c) 2020-2024, The Regents of the University of California,
```